src/parsers/xyz.js

```diff
@@ -1,11 +1,15 @@
+import Integer from "lodash/string";
 import _ from "underscore";
 import s from "underscore.string";
 
 import { Basis } from "../basis/basis";
 import { ConstrainedBasis } from "../basis/constrained_basis";
 import { Lattice } from "../lattice/lattice";
+// eslint-disable-next-line import/no-cycle
+import { defaultMaterialConfig } from "../material";
```

Comment:
I understand your comment above, but we should not be introducing circular dependencies with new code. It's usually a sign of not having a correct placement of the code